modules/view_generator/README.md

```diff
@@ -0,0 +1,20 @@
+# bigquery_view_generator
+
+Generates big query views without blacklisted field.s
```

Comment:
We should have a more robust README, that minimally links to this doc: https://cloud.google.com/bigquery/docs/authorized-views#:~:text=Giving%20a%20view%20access%20to,access%20to%20the%20underlying%20tables.

Also, typo.